README.md

```diff
@@ -1 +1,37 @@
-# INF 6420 - Intro to Web Development
+# INF6420 Project Hannah Shumway
+
+A site serving as a brief introduction to folklore.
+
+## Wireframes
+
+These are some basic layouts of how the website will hopefully look upon completion. 
+
+![Wireframe of Home Page](wireframes\wireframehome.jpg)
```

Comment:
Hi @HShumway use a `/` instead of the `\` to resolve the broken images.

Reply:
Okay I think I fixed it! Thank you so much, I copied the links on VSC without noticing the difference in the slashes, I'll keep an eye on that going forward!